zap/src/main/java/org/zaproxy/zap/extension/pscan/scanner/ScriptsPassiveScanner.java

```diff
@@ -121,15 +121,11 @@ private boolean appliesToCurrentHistoryType(ScriptWrapper wrapper, PassiveScript
         }
     }
 
-    /** @since TODO add version */
-    @Override
-    public AlertBuilder newAlert() {
-        return super.newAlert();
-    }
-
```

Comment:
Was this removed intentionally?

Reply:
Oops that shouldn’t have been an approve till this is discussed...

Reply:
Arg no, missed that :(